Sources/LiveViewNative/Coordinators/LiveSessionCoordinator.swift

```diff
@@ -585,22 +584,50 @@ extension LiveSessionCoordinator {
 
     private static func getTarget() -> String {
         #if os(watchOS)
-        return "watch"
+        return "watchOS"
         #elseif os(macOS)
-        return "mac"
+        return "macOS"
+        #elseif os(visionOS)
+        return "visionOS"
+        #elseif os(tvOS)
+        return "tvOS"
         #else
         switch UIDevice.current.userInterfaceIdiom {
         case .phone:
-            return "phone"
+            return "iOS"
         case .pad:
-            return "pad"
+            return "iPadOS"
         case .mac:
-            return "mac"
+            return "macCatalyst"
         case .tv:
-            return "tv"
+            return "tvOS"
+        case .vision:
+            return "visionOS"
         default:
-            return "unspecified"
+            return "unknown"
```

Comment:
What should the default be? It could be changed to `nil`.

Reply:
Let's go with `"unknown"`